extensions/amp-sidebar/1.0/test/test-component.js

```diff
@@ -19,6 +19,10 @@ import {Sidebar} from '../component';
 import {mount} from 'enzyme';
 
 describes.sandboxed('Sidebar preact component', {}, (env) => {
+  const isOpened = (sidebarElement) => {
+    return !sidebarElement.className.includes('unmounted');
+  };
+
```

Comment:
How the unit tests check whether the sidebar is opened or not must be updated throughout the test.